### DISCUSSION TITLE
How Can I Provide Data Through Chainbase?

### DISCUSSION THREAD

**Discussion by junyoungcross** [chainbase-labs/manuscript-core#85] [General]
I currently possess high-quality datasets covering 11 mainnets, including EVM-based networks. Therefore, I am interested in supplying data through Chainbase. Could you guide me on how I can provide my datasets via Chainbase? Additionally, I would like to know how the fees or compensation for data provision are determined.

**Comment by Liquidwe** [comment]
Hi @junyoungcross At this stage, we have only opened up the ability for users to generate data through manuscript, and we have not yet opened up the integration of user-defined datasets. However, we are very interested in your datasets. If you can build these data through manuscript, chainbase will have an incentive plan for developers.

**Reply by junyoungcross** [reply]
thanks :)
